workshops/views.py

```diff
@@ -1307,6 +1314,32 @@ def event_invoice(request, event_ident):
 
 
 @login_required
+def events_tag_changed(request):
+    """List events with metatags changed."""
+    events = Event.objects.active().filter(tags_changed=True)
+
+    assigned_to, is_admin = assignment_selection(request)
+
+    if assigned_to == 'me':
+        events = events.filter(assigned_to=request.user)
+
+    elif assigned_to == 'noone':
+        events = events.filter(assigned_to=None)
+
+    elif assigned_to == 'all':
+        # no filtering
+        pass
```

Comment:
I'm mostly thinking that if we don't recognize the value of 
'assigned_to', we should halt with an internal error.